larray/inout/pandas.py

```diff
@@ -291,9 +293,27 @@ def _series_to_group(series, axis):
     return LGroup(key=series.values, name=name, axis=axis)
 
 
-# ################################## #
-#    DATAFRAME <--> AXES, GROUPS     #
-# ################################## #
+def _metadata_to_series(metadata):
+    s = pd.Series(data=list(metadata.values()), index=metadata.keys(), name='')
+    s.index.name = 'METADATA'
+    return s
+
+
+def _series_to_metadata(series):
+
+    def _convert_value(key, value):
+        value = str(value)
```

Comment:
Why is this line needed? It seems weird to convert numeric types to string to convert them back on the next line to numeric?